app/models/application_record.rb

```diff
@@ -1,3 +1,8 @@
 class ApplicationRecord < ActiveRecord::Base
   self.abstract_class = true
+
+
+  def esential
+    aplication = 'tranction'
```

Comment:
__[development][Misspell]__

"aplication" is a misspelling of "application"


<!-- Comment by [Sider](https://sider.review) -->